backend/tests/test_func_login_with_db.py

```diff
@@ -1,14 +1,11 @@
 from tools.create_db_for_tests import create_test_app_with_db
-from tools.work_with_db import get_psw_from_db
-import db.models as models
+from tools.work_with_db import get_psw_from_db, push_email_psw_to_db
 
 
 def test_get_log_psw_from_db():
     log = 'admin'
     psw = 'Pyth0n'
     create_test_app_with_db()
-    test_admin = models.AdminInfo(email=log, psw=psw)
-    models.db.session.add(test_admin)
-    models.db.session.commit()
+    push_email_psw_to_db(log, psw)
```

Comment:
After we merge current open PRs it would make sense check if it's possible make this replacement in other places.

Reply:
it's a great idea